script/build-and-upload

```diff
@@ -94,3 +94,6 @@ set -x
 
 cabal upload --publish "$TGZ_LOC"
 cabal upload --publish --documentation "$DOCS_LOC"
+
+echo "Tagging $PACKAGE_VERSION"
+git tag "$PACKAGE_VERSION"
```

Comment:
This is going to do a `git push --tags` as well once I’ve tested it out in anger by pushing `tree-sitter` up.